shapefile.py

```diff
@@ -201,6 +201,11 @@ def __init__(self, shapeType=NULL, points=None, parts=None, partTypes=None):
 
     @property
     def __geo_interface__(self):
+
+        """  Case parts or points is empty """
+        if not self.parts or not self.points:
+            return False

```

Comment:
False is a valid value for geojson data structure. (https://geojson.org/geojson-spec.html#geometry-objects) I did some tests with geojson that contains a false value in geometry fields in GIS tools, and i not found erros.